python_modules/libraries/dagster-components/dagster_components/core/defs_module.py

```diff
@@ -172,6 +206,7 @@ def from_path(path: Path) -> Optional["SubpackageDefsModuleDecl"]:
     def load(self, context: ComponentLoadContext) -> SubpackageDefsModule:
         return SubpackageDefsModule(
             path=self.path,
+            defs_config=ResolvedDefsConfig.from_path(self.path / "defs.yaml"),
```

Comment:
Curious if we want to consider a `properties.yaml` taken from dbt's play book.

https://docs.getdbt.com/reference/model-configs

<img width="966" alt="image" src="https://github.com/user-attachments/assets/c2f33f3d-dbe8-4095-9a01-2b4728ec2b54" />

When I see a `defs.yaml` I feel like it would be a definition, and not a default configuration for definitions.

Reply:
The mental model here IMO is that `defs.yaml` is just giving instructions to the system about how load the current definitions module. We can use this instead of `component.yaml` as well FWIW. Components are just definitions factories are their core. So in that framework you can justify as "you are a factory. You take all the definitions objects loaded in your children and post process them"